src/component/dataZoom/InsideZoomView.js

```diff
@@ -114,11 +114,15 @@ var roamHandlers = {
         var directionInfo = getDirectionInfo[coordSysName](
             null, [e.originX, e.originY], axisModel, controller, coordInfo
         );
+
+        // diff=0 will freezing dataZoom
+        var rangeDiff = (range[1] - range[0]) || 0.1;
```

Comment:
I don't think it's a good idea to use an arbitrary number for the default range.
Maybe we could provide an option for the user and set the default value to be 0.1.

Reply:
provide an option named `dataZoom.minRangeDiff` and its default value is `0.1`, right?

`range` is an internal concept, I don't know if `rangeDiff` is a good name.